scm/scm_test.go

```diff
@@ -210,5 +326,20 @@ func TestHelperProcess(t *testing.T) {
 		os.Exit(0)
 	case "FAILED_PULL":
 		os.Exit(1)
+	case "SUCCESS_TO_KILL":
+		if subcmd == "command" {
```

Comment:
Would it be easier to understand if it was `sleep`?

Reply:
thx.
I fixed.